frontend/pkg/frontend/ocm.go

```diff
@@ -662,15 +661,20 @@ func (f *Frontend) BuildCSExternalAuth(ctx context.Context, externalAuth *api.HC
 
 	}
 
-	for _, t := range externalAuth.Properties.Clients {
```

Comment:
This implementation is present in a few other places and I'll fix them in a subsequent PR i.e. Taints in Nodepools

Reply:
Good catch.